CONTRIBUTING.md

```diff
@@ -86,9 +86,9 @@ Fix #XYZ, SHORT_DESCRIPTION
 
 LONG_DESCRIPTION (optional)
 ```
-5. Sign and email the appropriate Contributor License agreement down below to GSFC-SoftwareRelease@mail.nasa.gov and copy cfs-program@lists.nasa.gov.
-    - Corporate Contributor License agreement : https://github.com/nasa/cFE/blob/main/docs/GSC_18128_Corp_CLA_form_1219.pdf
-    - Individual Contributor License agreement : https://github.com/nasa/cFE/blob/main/docs/GSC_18128_Ind_CLA_form_1219.pdf
+#### Sign and email the appropriate Contributor License agreement down below to GSFC-SoftwareRelease@mail.nasa.gov and copy cfs-program@lists.nasa.gov
```

Comment:
Keep the heading short to make navigation easier `Contributor License Agreement (CLA)`

Then have the first sentence in the section be the "Sign and email..." finally edit the "down below" part to just say "below"